### ISSUE TITLE
Apps won't build against API 34 w/1.10.0-alpha03

### ISSUE THREAD

**Issue by fat-tire** [material-components/material-components-android#3443]
Today Android's API 34 went [stable](https://android-developers.googleblog.com/2023/06/android-14-beta-3-and-platform-stability.html).

However, when compling with `compileSdk=34` we see:
```
An issue was found when checking AAR metadata:

  1.  Dependency 'com.google.android.material:material:1.10.0-alpha03' requires libraries and applications that
      depend on it to compile against codename "UpsideDownCake" of the
      Android APIs.

      :app is currently compiled against android-34.

      Recommended action: Use a different version of dependency 'com.google.android.material:material:1.10.0-alpha03',
      or set compileSdkPreview to "UpsideDownCake" in your build.gradle
      file if you intend to experiment with that preview SDK.
```

Can you make an alpha-04 or whatever?  Or is there a workaround in `build.gradle.kts` to ignore this for now?

**Comment by hunterstich** [comment]
Hi @fat-tire,

This should now be fixed in [1.10.0-alpha04](https://github.com/material-components/material-components-android/releases/tag/1.10.0-alpha04). Closing but let me know if you run into any issues with the latest release.

**Comment by fat-tire** [comment]
Thanks so much!
